### ISSUE TITLE
How run the game Heroes of Storm from Bilzzard?

### ISSUE THREAD

**Issue by ZeroCool22** [mhasanjb/ForceBindIP-Gui#3]
I have 2 Adapters in my PC Ethernet Cards, and 2 different inet connections (different ISP's).

So i want to use 1 connection for stream and play in the other the game.

But Heroes of Storm seems to be some tricky to get it working since the game have a Launcher (Battlenet) and it call the game to start.

**- Launcher process and exe:**

![Battlenet location](https://user-images.githubusercontent.com/13344308/92337327-2c1f8f00-f07f-11ea-8965-a90b004f4486.png)

**- Game itself, process and launch location:**

![HOTS Location](https://user-images.githubusercontent.com/13344308/92337339-3f325f00-f07f-11ea-92ae-6e96eb6364cb.png)

**Comment by mhasanjb** [comment]
If you don't know how to fix it,
I have a trick for u to do. 
Just change order of your "primary" & "secondary" internet connections.
For example, 
You may want to use your "Connection 001" for stream & "Connection 002" for Game.
So you can change "Connection 001" and "Connection 002" priority. and then just opening your StreamLabs OBS ( or whatever you use ) with that "secondary" connection.
Follow this steps: 
1- Go to control panel and then network connections (Control Panel\All Control Panel Items\Network Connections)
2- Right click on your connections
3- Properties
4- Internet Protocol Version 4 (TCP/IPv4) -> Properties
5- Advanced
6- Uncheck "Automatic metric"
7- Write a number for order ( example 1 )
8- Ok and exit.
do the same for your another Connection and make the order to 2
then you can see your main internet connection will change to that one who has "1" metric.
after that you can go to GUI and open "StreamLab" with your "second" (metric 2) connection

So you really don't need to open your game with "GUI", you only need to open your StreamLAB  with GUI

TLDR; 
Main connection (metric 1) = Game
Second connection (metric 2) = Stream

**Images
![001](https://user-images.githubusercontent.com/39640448/92353252-1f3d8280-f0f5-11ea-9e3c-1a735a6ae260.png)

![002](https://user-images.githubusercontent.com/39640448/92353257-22387300-f0f5-11ea-86a9-66625c2622e4.png)

![003](https://user-images.githubusercontent.com/39640448/92353451-9b37ca80-f0f5-11ea-9b12-6801d892453b.png)

**Comment by ZeroCool22** [comment]
Yeah, i did that, thx.
